tasks.py

```diff
@@ -295,3 +295,35 @@ def write_file(full_path: Path, content: str):
                 full_path=docs_sub_dir.joinpath(md_filename),
                 content=template.format(name=basename, py_path=py_path),
             )
+
+
+@task(help={"url": "URL of the file to validate"})
+def run_swagger_validator_on_url(_, url=None):
```

Comment:
I think this name could be shorter? Like `swagger_validation`, `swagger_validator` or `run_swagger_validator` or something?

Reply:
Yep, done